### PR TITLE
Add node 10 for RHOAR boosters

### PR DESCRIPTION
**Pull request by lholmquist**
This adds node 10 for the "RHOAR" production boosters.

I'm keeping this separate since this might need some more release notes and such before it is merged

### CODE REVIEW

**Comment by gastaldi** [comment]
[test]

**Comment by Ladicek** [comment]
Should probably be marked `WIP` to avoid accidental merge before Node.js 10 is actually releasd.

File: metadata.yaml [hunk 1]
```diff
@@ -103,6 +103,8 @@ runtimes:
     name: 8.x (Community)
   - id: redhat
     name: 8.x (RHOAR)
+  - id: v10-redhat
```
Comment (lance) [inline metadata.yaml]: Is this because the ID for `8.x` was just `redhat` and we need to start distinguishing versions?

Comment (Ladicek) [reply metadata.yaml]: Right. Actually depends on if you want to keep 8.x boosters available together with 10.x?

E.g. with WildFly Swarm (and Vert.x too, I think), we're just keeping the latest. Spring Boot keeps "current" and "previous".

Comment (lholmquist) [reply metadata.yaml]: @Ladicek beat me to it. but yes.

**Comment by lholmquist** [comment]
@gastaldi so i got @lance 's approval review, but how do i merge this then?

**Comment by gastaldi** [comment]
[test]

**Comment by gastaldi** [comment]
@lance  can you approve again? I think this PR was created before I fixed the Codeowners bug

**Comment by gastaldi** [comment]
@lholmquist there, you should see the merge button now

**Pull request merged**